metric/system/process/process_linux_test.go

```diff
@@ -60,7 +61,11 @@ func TestFetchOtherProcessCgroup(t *testing.T) {
 	assert.NoError(t, err, "Init")
 
 	evts, _, err := testConfig.Get()
-	require.NoError(t, err)
+	if err != nil {
+		assert.True(t, errors.Is(err, NonFatalErr{}))
```

Comment:
[Blocker]
It'd be better to have a message on the `assert.True`. Getting a "false is not true" as the error message ins't much of a help.

Also, in its current state we'd lose the error, making it hard to understand the issue.
same for the other tests.
```suggestion
		assert.Truef(t, errors.Is(err, NonFatalErr{}, "err isn't NonFatalErr. err: %v", err))
```